lib/active_merchant/billing/gateways/realex.rb

```diff
@@ -472,6 +472,8 @@ def message_from(response)
         else
           message = DECLINED
         end
+
+        message
```

Comment:
It shouldn't matter because `x = y` returns `y`.

If this is for readability, a better improvement might be to simplify it to just:
```ruby
case response[:result]
when "00" then SUCCESS
when "101" then response[:message]
...
```

Reply:
Nice catch, Julian. It looks like Bruno made this change to [suppress a warning](https://buildkite.com/waysact/active-merchant/builds/481#01894e35-f045-4140-bf24-c651df33d048/105-189). Your simplification makes sense to me and upstream has [made a similar change](https://github.com/activemerchant/active_merchant/commit/4abd2f0db48a6314ab16906525efb34f8b7e6bc5#diff-8f17a82e2a38e71c76ad882d3c1a6a684606e41ba57cf9d484365a3f9b14f523). Since Bruno is busy with the audit I've taken the liberty of [applying that change to this PR](https://github.com/waysact/active_merchant/pull/247/commits/73f97675f87efcdccab3f48a4261ad1e2d66d35c).